lib/config.ts

```diff
@@ -6,8 +6,10 @@
  */
 
 import { parsePageId } from 'notion-utils'
-import { getSiteConfig, getEnv } from './get-config-value'
-import { PageUrlOverridesMap, PageUrlOverridesInverseMap } from './types'
+import posthog from 'posthog-js'
```

Comment:
why not `import { Config as PosthogConfig } from 'posthog-js'`?

Reply:
<img width="749" alt="image" src="https://user-images.githubusercontent.com/3807458/161718802-1678b74a-5b2b-4926-84cc-ae4be012d3bd.png">
It doesn't work because it's encapsulated within a namespace.

I tried `import Config from 'posthog-js'` but it doesn't work well, autocompletion fails.

I suggest keeping it the way it is, their TS types aren't the easier to use.

Reply:
sounds good :)